### PR TITLE
Fix test_primary_failure test case

### PR DESCRIPTION
**Pull request by fanfuxiaoran**
add plpython_version variable in test sql.
On gpdb6, plpython_version is plpythonu, on gpdb7, it's plpython3u

### CODE REVIEW

**Comment by zhrt123** [comment]
Should we use the name `test_primary_failure.in.sql`, which is the same as `test_rejectmap.in.sql`?

**Comment by fanfuxiaoran** [comment]
> Should we use the name `test_primary_failure.in.sql`, which is the same as `test_rejectmap.in.sql`?

personally, I prefer this way. @beeender , how about you?

File: tests/regress/sql/test_primary_failure.sql [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
+\gset
+select CASE WHEN :gp_server_version_num < 70000 THEN 'plpythonu' else 'plpython3u' end plpython_version;
+\gset
+-- end_ignore
+
 CREATE SCHEMA ftsr;
 SELECT diskquota.set_schema_quota('ftsr', '1 MB');
 SET search_path TO ftsr;
-create or replace language plpythonu;
+create or replace language :plpython_version;
```
Comment (beeender) [inline tests/regress/sql/test_primary_failure.sql:12]: Let's use the way we introduced in #304  , so we can save lines 1-7.

Let me know if you don't have time to do that, I can help.

Comment (fanfuxiaoran) [reply tests/regress/sql/test_primary_failure.sql]: Done

**Review by beeender — APPROVED** [review]

**Review by beeender — CHANGES_REQUESTED** [review]
Wait,
`diskquota_schedule7` should be deleted now. And the `EXPECTED_DIR_SUFFIX` in the `tests/CMakeLists.txt` can be removed as well.

**Comment by fanfuxiaoran** [comment]
> Wait, `diskquota_schedule7` should be deleted now. And the `EXPECTED_DIR_SUFFIX` in the `tests/CMakeLists.txt` can be removed as well.

Ok, I'll fix it by the way.

**Review by beeender — APPROVED** [review]

**Pull request merged**